Sources/SentryCrash/Recording/Tools/SentryCrashFileUtils.c

```diff
@@ -394,6 +394,10 @@ sentrycrashfu_removeFile(const char *path, bool mustExist)
 bool
 sentrycrashfu_deleteContentsOfPath(const char *path)
 {
+    if (path == NULL) {
+        return false;
+    }
```

Comment:
`m`: How is that change related to this PR?

Reply:
Ah, I forgot to comment on this one. I added a call to `sentrycrash_deleteAllReports` to `clearTestState` (which was done in some of the crash handler unit tests, so I added it here to apply to all tests) but then for test cases that had never written a crash report, this would crash with a NPE. I figured this fix would also just be a nice safeguard to have in production too.